app/internal/page/donor/restrictions.go

```diff
@@ -29,15 +29,15 @@ func Restrictions(tmpl template.Template, lpaStore page.LpaStore) page.Handler {
 		}
 
 		if r.Method == http.MethodPost {
-			form := readRestrictionsForm(r)
```

Comment:
you could have left these as `form`, it only becomes an issue when you have a variable called `form` in scope _and_ also want to use something from the package

Reply:
Cool - I think I was running into issues in tests and just did a batch update. I do prefer the more descriptive var name so I'll see how reverting to `form` goes.